cmd/terramate/cli/run.go

```diff
@@ -455,9 +455,11 @@ func (c *cli) runAll(
 			}
 
 			err = errs.AsError()
-			if err != nil && !continueOnError {
+			if err != nil {
 				logger.Info().Msg("interrupting execution of further stacks")
```

Comment:
This will say it cancels further stacks, but it doesn't in case of continueOnError.

Reply:
Fixed